src/pages/SearchPage/Heatmap/Heatmap.style.js

```diff
@@ -23,104 +23,16 @@ const TopRow = styled.div`
   justify-content: flex-end;
 `;
 
-const HoursContainer = styled.div`
-  box-sizing: border-box;
-  display: flex;
-  height: 100%;
-  width: auto;
-  color: #757575;
-  font-size: 14px;
-  font-weight: 500;
-  box-shadow: 0px -1px 2px -2px rgba(0,0,0,0.1);
-  &:first-child {
-    border-left: 1px solid #f3f3f3;
-  }
-  &:last-child {
-    border-right: 1px solid #f3f3f3;
-  }
-`;
-
-const Hour = styled.div`
-  box-sizing: border-box;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 53px;
-  width: 80px;
-  background: linear-gradient(180deg, #fefefe 0%, #e9e9e9 100%);
-  border-top: 1px solid #f3f3f3;
-`;
-
 const BottomRow = styled.div`
   display: flex;
   flex-direction: rows;
   height: 280px;
```

Comment:
You don't need this height since the height of 40px is already defined for the Info components